tmpl/util.go

```diff
@@ -188,22 +189,12 @@ func (f *tmplFuncs) fullyQualified(typePath string) string {
 
 // resolvePkgPath resolves the named protobuf package, returning it's file
```

Comment:
Minor typo here, "returning it's file path" should be "returning its file path".